doc/specs/vulkan/chapters/initialization.txt

```diff
@@ -30,41 +30,32 @@ Loaders are encouraged to export function symbols for all other core
 the core Vulkan commands have no need to use fname:vkGetInstanceProcAddr.
 
-Function pointers to commands that don't operate on a specific instance can:
-be obtained by using this command with pname:instance equal to `NULL`. The
-following commands can: be accessed this way:
-
-  * flink:vkEnumerateInstanceExtensionProperties
-  * flink:vkEnumerateInstanceLayerProperties
-  * flink:vkCreateInstance
-
-If pname:instance is a valid sname:VkInstance, function pointers to any
-commands that operate on pname:instance or a child of pname:instance can: be
-obtained. The returned function pointer must: only be called with a
-dispatchable object (the first parameter) that is a child of pname:instance.
-
-If pname:pName is not the name of a core Vulkan command, or is an
-extension command for any extension not supported by any available layer or
-implementation, then fname:vkGetInstanceProcAddr will return `NULL`.
-
-ifdef::editing-notes[]
-[NOTE]
-.editing-note
-====
-(Jon, Bug 14886 / Gitlab issue 4) The WSI group tentatively agreed that the
-WSI extensions were special, and should get static entry points in link
-libraries and prototypes in +vulkan.h+, while future extensions would have
-to be dynamically loaded. If this decision is upheld by the group as a
-whole, it would probably be encoded in the previous paragraph, in the WSI
-extensions branch of the Specification.
-
-However, this decision hasn't been fully signed off on by the entire Vulkan
-WG yet AFAIK. Note that implementations typically won't support many of the
-WSI extensions, so ``static entry points'' doesn't relieve apps of the
-neccessity of runtime enabling and testing of each extension before using
-it.
-====
-endif::editing-notes[]
+The table below defines the various use cases for fname:vkGetInstanceProcAddr
+and expected behavior ("fp" is function pointer) for each case. Each rule
+is tested in the order below.
+
+.vkGetInstanceProcAddr behavior
+[width="80%",options="header",align="center"]
+|=====
+| pname:instance | pname:pName | behavior
+| invalid instance | * | undefined
+| * | VK_NULL_HANDLE | undefined
+| VK_NULL_HANDLE | flink:vkEnumerateInstanceExtensionProperties | return fp
+| VK_NULL_HANDLE | flink:vkEnumerateInstanceLayerProperties | return fp
+| VK_NULL_HANDLE | flink:vkCreateInstance | return fp
+| VK_NULL_HANDLE | * (not any row/case above) | return VK_NULL_HANDLE
+| instance | core Vulkan command | return fp (1)
+| instance | enabled instance extension command for pname:instance | return fp (1)
+| instance | available device extension command for pname:instance | return fp (1,2)
+| instance | * (not any row/case above) | return VK_NULL_HANDLE
+|=====
+
+1) The returned function pointer must: only be called with a dispatchable object
+(the first parameter) that is pname:instance or a child of pname:instance (
+    i.e. sname:VkPhysicalDevice, sname:VkDevice, sname:VkQueue, or
+    sname:VkCommandBuffer).
```

Comment:
> (Still am inclined, the list should be in the chapter where "dispatchable handle" term is defined instead)

That's tough. Not sure there is a "best" place that doesn't duplicate the info, which I don't think we want to do. Of the two places, definition of kinds of handles Vulkan can generate, and description of how to "initialize" Vulkan, I think it fits the "initialize" chapter better.